template.tpl

```diff
@@ -2398,3 +2398,5 @@ setup: |-
 ___NOTES___
 
 Created on 16/10/2023, 09:46:49
+
+
```

Comment:
i don't understand this part of the code

Reply:
GTM seems to export with a couple of extra lines on the bottom now, for what reason I don't know.